L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -527,9 +520,21 @@ void HGCalHistoSeedingImpl::findHistoSeeds(const std::vector<edm::Ptr<l1t::HGCal
     if (seedingSpace_ == RPhi) {
 
       navigator_ = Navigator(nBins1_, Navigator::AxisType::Bounded, nBins2_, Navigator::AxisType::Circular); 
-      
+     
+      for(unsigned int nb1 = 0; nb1 < nBins1_; nb1++) {
+	binsSumsHisto_[it][nb1] = binsSumsHistoN_[nb1+nbins1_counter];
+      }
+      nbins1_counter += nBins1_;	
+	
+      if (seedingAlgoType_.find("Histo") != std::string::npos &&
+	  nBins1_ != binsSumsHisto_[it].size()) {
+	throw cms::Exception("Inconsistent bin size")
+	  << "Inconsistent nBins_X1_histo_multicluster ( " << nBins1_ << " ) and binSumsHisto ( " << binsSumsHisto_[it].size()
+	  << " ) size in HGCalMulticlustering\n";
+      }
```

Comment:
Consistency checks would be better in the constructor. There is no need to perform the check for each event.